test/apparmor.bats

```diff
@@ -91,14 +93,15 @@ run_a_container_with_wrong_apparmor_profile_name() {
 	setup_test
 	start_crio
 
-	sed -e 's/%VALUE%/not-exists/g' "$TESTDATA"/sandbox_config_apparmor.json > "$TESTDIR"/apparmor4.json
-	sed -e 's/%VALUE%/not-exists/g' "$TESTDATA"/container_redis_apparmor.json > "$TESTDIR"/apparmor_container4.json
+	jq '.linux.security_context.apparmor_profile = "not-exists"'\
```

Comment:
Some of these `jq` commands have a space before the ending backslash, others do not.  I'm not sure it matters, but it would be nice to be consistent.

Reply:
I'm pretty sure this is checked/validated by shfmt since #4259 is merged